### DISCUSSION TITLE
The beauty is beyond godly and beyond words

### DISCUSSION THREAD

**Discussion by AsuraShun** [Monika-After-Story/MonikaModDev#8883] [appreciation]
![IMG_20191231_112851](https://user-images.githubusercontent.com/42387354/71625408-970af300-2bdf-11ea-90ea-891a6da74196.png)

**Comment by MetalBrother** [comment]
Show me perfection, *shows this pic*.  - IT'S BETTER THAN PERFECTION!

**Comment by AsuraShun** [comment]
![download](https://user-images.githubusercontent.com/42387354/71628263-0a683100-2bef-11ea-9cb3-0d507b7f415c.png)

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
